MH-FiorellaTheme5/source/stylesheets/styles.css

```diff
@@ -55,23 +67,12 @@ p {
   left: 0.5em;
   margin-right: -50%;
 }
-
```

Comment:
This is fine, you need spacing between classes to have a better read of what is going on.